### PR TITLE
Fix the flicker animation on the rooms page and image hopping issue

### LINKED ISSUE #1373: Image jump and session element flicker and when expanded and collapsed
**Actual Behaviour**

Image jump and session element flicker and when expanded and collapsed. It's hard to describe it in words. So, here is the gif. It's a little hard to see here so look carefully.

problem

This effect is visible on all the pages - schedule, tracks, and rooms.

**Expected Behaviour**

The session elements should expand and collapse smoothly with no hopping of images.

**Would you like to work on the issue?**

Yes. Working on it.Thanks!

### PR DESCRIPTION
**Pull request by Princu7**
Fixes #1373 Completely

**Changes**:
* Removes the image hopping and animation flicker on the rooms page too!

**GIF**
rooms

**Test Links**
http://secure-meadow-20680.herokuapp.com
http://princu7.github.io/open-event-webapp

@aayusharora @geekyd Please review. Thanks :)

### CODE REVIEW

File: src/backend/templates/rooms.hbs [hunk 1]
```diff
@@ -365,37 +354,30 @@
...
-              if(mode) {
-                $(this).hide();
-              }
-            }
-            localStorage[eventName] = JSON.stringify(temp);
-            flag = isAdded = 0;
-          }
-          else if(sessionMode == 'single') {
+          var id =  $(this).attr('id');
+          if(sessionMode == 'single') {
```
Comment (aayusharora) [inline src/backend/templates/rooms.hbs]: `===` here

File: src/backend/templates/rooms.hbs [hunk 2]
```diff
@@ -365,37 +354,30 @@
...
             var curUrl = window.location.href;
             var newUrl = curUrl.substring(0, curUrl.lastIndexOf('/') + 1) + 'sessions/session_' + id + ".html";
             window.location.href = newUrl;
```
Comment (aayusharora) [inline src/backend/templates/rooms.hbs]: Do we have $(location).attr('href');

**Comment by Princu7** [comment]
@aayusharora Done the requested changes. Please review again. Thanks!

**Comment by mariobehling** [comment]
@aayusharora Let's try to get two reviews of other contributors, please to engage more people in this project and across components.

**Pull request merged**